spec/models/proceeding_type_spec.rb

```diff
@@ -44,4 +44,21 @@
       end
     end
   end
+
+  describe '#domestic_abuse?' do
+    let(:proceeding_type) { create :proceeding_type, ccms_matter: matter }
+    context 'domestic abuse' do
+      let(:matter) { 'Domestic Abuse' }
+      it 'returns true' do
+        expect(proceeding_type.domestic_abuse?).to be true
+      end
+    end
+
+    context 'something else' do
```

Comment:
the context could benefit from being more descriptive, e.g `when the matter is not domestic_abuse`